config.go

```diff
@@ -690,6 +675,11 @@ func loadConfig() (*config, []string, error) {
 		log.Warnf(str, funcName)
 	}
 
+	if cfg.VotingWalletExtPub != "" {
+		str := "%s: Config VotingWalletExtPub is deprecated and has no effect. Please remove from your config file."
```

Comment:
I'd also explicitly state that it will be removed in a future release, even though that's the definition of deprecated, and it should be removed to avoid startup errors in the future.